#!/usr/bin/env python
PACKAGE = "dcsl_high_level_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("go_to_initial_states", bool_t, 0, "Turn on waypoint control for initial positions", False)
gen.add("run_control_law", bool_t, 0, "Turn on control law for all robots", False)

exit(gen.generate(PACKAGE, "dcsl_miabot_high_level_control.py", "dcsl_miabot_high_level_control_config"))
